deploy/grafana-dashboard/utils/color_mapping.py

```diff
@@ -19,3 +23,13 @@ def color_mapping(name, color):
 
 
 grey_all_mapping = color_mapping("all", "#4c4b5a")
+color_mapping_environments = {}
+colours = ["yellow", "blue", "orange", "red", "purple",
+           "pink", "brown", "black", "white", "gray",
+           "gold", "super-light-red", "dark-red", "dark-orange", "super-light-yellow",
+           "super-light-green", "dark-green", "dark-blue", "super-light-purple", "super-light-blue"]
```

Comment:
What do you think of using colors from the [R ColorBrewer](https://r-graph-gallery.com/38-rcolorbrewers-palettes.html)?

```suggestion
colours = [
            "#E41A1C", "#377EB8", "#4DAF4A", "#984EA3", "#FF7F00", "#FFFF33", "#A65628", "#F781BF", "#999999", # R ColorBrewer Set1
            "#8DD3C7", "#FFFFB3", "#BEBADA", "#FB8072", "#80B1D3", "#FDB462", "#B3DE69", "#FCCDE5", "#D9D9D9", "#BC80BD", "#CCEBC5", "#FFED6F" # R ColorBrewer Set3
]
```